.travis.yml

```diff
@@ -1,11 +1,17 @@
 language: python
 dist: xenial
-python:
-  - "3.6"
-  - "3.7"
 
 install:
   - pip install -r dev-requirements.txt
 
-script:
-  - python -m pytest test_esss_jenkins.py
+jobs:
+  include:
+    - script: python -m pytest test_esss_jenkins.py
+      python: "3.6"
+      name: "Python 3.6"
+    - script: python -m pytest test_esss_jenkins.py
+      python: "3.7"
+      name: "Python 3.7"
+    - script: pre-commit run --all-files --show-diff-on-failure
+      python: "3.6"
```

Comment:
Don't we need this for Python 3.7 as well?

Reply:
You mean `pre-commit`? Hardly, 3.6 and 3.7 syntax are identical. :)

Reply:
Ok :)